etc/kayobe/stackhpc-monitoring.yml

```diff
@@ -19,3 +19,16 @@ stackhpc_enable_os_capacity: true
 # Whether TLS certificate verification is enabled for the OpenStack Capacity
 # exporter during Keystone authentication.
 stackhpc_os_capacity_openstack_verify: true
+# Redfish exporter
+
+# Whether the redfish exporter is enabled.
+stackhpc_enable_redfish_exporter: false
+
+# Credentials
+redfish_exporter_default_username: "{{ ipmi_admin_user }}"
+redfish_exporter_default_password: "{{ ipmi_admin_password }}"
```

Comment:
Why are we using ipmi_admin_user / ipmi_admin_password rather than ipmi_username / ipmi_password? The latter are the standard variables in kayobe.

Reply:
Changed.